packages/cozy-procedures/src/redux/personalDataSlice.js

```diff
@@ -57,10 +58,10 @@ const personalDataSlice = createSlice({
 
       state.data = {
         ...state.data,
-        salary: summedStats.income,
-        additionalIncome: summedStats.additionalIncome,
-        propertyLoan: summedStats.mortgage,
-        creditsTotalAmount: summedStats.loans
+        salary: round(summedStats.income, 2),
```

Comment:
i would like the `2` to be in a configuration object. It will be different for different currencies

Reply:
Extracting round to a util roundCurrencyAmount would be alright ?

Reply:
with `roundCurrencyAmount( amount, currency)` ? 👌